app/presenters/publishing_api_presenters/item.rb

```diff
@@ -33,13 +33,28 @@ def content
   end
 
   def links
-    {}
+    PublishingApiPresenters::LinkExtractor.new(item).extract(filter_links)
   end
 
 private
 
   attr_accessor :item
 
+  def filter_links
```

Comment:
I think it's better to leave this default out. Otherwise we'll still won't know if every format has defined it's own links yet.